pytrickle/client.py

```diff
@@ -130,31 +216,62 @@ async def _ingress_loop(self):
                     logger.info("Stopping ingress loop due to error or stop signal")
                     break
                 
+                # Normalize timestamps to start from 0 for each stream type
```

Comment:
Rewriting timestamps seems sketchy.  How can this happen when we are getting decoded frames to process?

I think this kind of logic is better in the encoder as well if it is needed.